### DISCUSSION TITLE
Q: Vercel部署完成并绑定域名后，使用域名访问不了，而vercel默认链接可以访问正常 || Q: After Vercel is deployed and bound to the domain name, it cannot be accessed using the domain name, but the default link of vercel can be accessed normally.

### DISCUSSION THREAD

**Discussion by freedomgod** [walinejs/waline#2102] [Q&A] [answered]
### Describe the bug | 问题描述

我很早以前使用vercel + leancloud的方式成功部署了waline（在vercel绑定了域名），且能够正常使用，但前阵子突然访问不了，F12发现一个警告：
```
crbug/1173575, non-JS module files deprecated.
```
可以在<a href="https://waline.whfree.top/" target="_blank">链接</a>查看。
奇怪的是，使用域名访问不了，但vercel提供的默认访问链接<a href="https://hexo-waline-comment-seven.vercel.app/" target="_blank">链接</a>是能够正常访问的，这应该是能说明不是服务端的问题吧。

但仍不确定是前端的问题还是服务部署的问题，于是我首先重新把服务端部署了一遍，也没有用。然后我在阿里云的域名解析处检查一番，确认域名解析是没有问题的。所以来此求助，希望大佬们能够给予帮助，谢谢！

另：还想问的是vercel部署教程中，部署完成后默认是中文，但我部署后为啥默认是英文？

### Website URL | 问题网站

waline.whfree.top

### Where your waline deploy? | 服务部署在哪里？

Vercel (Default)

### Where your comment data store? | 数据存储在哪里？

LeanCloud(https://leancloud.app)

---
### Describe the bug | Problem description

I successfully deployed waline using vercel + leancloud a long time ago (the domain name was bound to vercel), and it can be used normally. However, I suddenly couldn't access it a while ago. F12 found a warning:
```
crbug/1173575, non-JS module files deprecated.
```
It can be viewed at <a href="https://waline.whfree.top/" target="_blank">link</a>.
The strange thing is that I can't access it using the domain name, but the default access link provided by vercel is <a href="https://hexo-waline-comment-seven.vercel.app/" target="_blank">link</a> It can be accessed normally, which should mean that it is not a server-side problem.

But I was still not sure whether it was a front-end problem or a service deployment problem, so I first redeployed the server, but it didn't work. Then I checked at Alibaba Cloud's domain name resolution office and confirmed that there was no problem with the domain name resolution. So I come here for help, I hope you guys can help me, thank you!

Another thing: I also want to ask: in the vercel deployment tutorial, the default is Chinese after the deployment is completed, but why is it the default is English after I deploy?

### Website URL | Problem website

waline.whfree.top

### Where your waline deploy? | Where is the service deployed?

Vercel (Default)

### Where your comment data store? | Where is the data stored?

LeanCloud(https://leancloud.app)

**Comment by lizheming** [comment]
感觉和你的域名有点关系，可以检查下相关的配置，目前我直接访问你的域名是没办法访问的

![image](https://github.com/walinejs/waline/assets/424491/040c2a74-e2c1-4f54-938a-804f97226dc1)

当然 Vercel 的域名也是没办法访问的，这个是已知的

![image](https://github.com/walinejs/waline/assets/424491/eb16f2fb-2cf8-4755-9826-17e63de69922)



---
It feels like it has something to do with your domain name. You can check the relevant configuration. Currently, I can’t access your domain name directly.

![image](https://github.com/walinejs/waline/assets/424491/040c2a74-e2c1-4f54-938a-804f97226dc1)

Of course, Vercel’s domain name cannot be accessed. This is known.

![image](https://github.com/walinejs/waline/assets/424491/eb16f2fb-2cf8-4755-9826-17e63de69922)

**Reply by freedomgod** [reply]
多谢，我发现只有开梯子才能访问。我刚刚也在相关群里看到有人说vercel被墙了，应该跟这个有关，好多博主的评论都显示不了🥲。
现在打算用阿里云函数计算部署，但是有个报错，好像是说代码大小超过50M需要再上传到OSS存储服务，教程中部署好像不需要？
```
✔ Checking Function Waline exists
[2023-10-02 15:29:56] [WARN] [FC-DEPLOY] - Fc tried to create bucket:waline in region:cn-beijing failed.
✖ Checking Service, Function (22.71s)
✖ Project framework failed to execute:

ERROR:

Please provide existed ossBucket under your account when code size is greater than 50M.

TraceId:     00155d0007001696231572343
Environment: @serverless-devs/s: 2.2.0, core: 0.1.64, s-home: C:\Users\WWH\.s, win32-x64, node-v19.7.0
Documents:   https://www.serverless-devs.com
Discussions: https://github.com/Serverless-Devs/Serverless-Devs/discussions
Issues:      https://github.com/Serverless-Devs/Serverless-Devs/issues
Regsitry:    https://registry.serverless-devs.com
```

![aliyun-fc-question](https://github.com/walinejs/waline/assets/59652488/69275b34-5124-4ede-9cb1-c0a2406d2d51)

**Reply by kankezhiyan** [reply]
请问老哥这个阿里部署的问题有解决办法了吗？我也卡在这一步了。

---
Bro, do you have a solution to this Alibaba deployment problem? I'm stuck at this step too.

**Reply by freedomgod** [reply]
我后面没有选择阿里方法部署了，这个我只知道大概的意思是什么，我猜就是可能原先waline占用空间少于50M，但随着更新空间就大于50M了，阿里云函数部署的教程没有更新考虑到这一步。具体怎么操作我就不太清楚了。主要是后面没时间折腾了，就放弃这种方式了。

---
I did not choose the Alibaba method to deploy later. I only know the general meaning of this. My guess is that the original space occupied by waline is less than 50M, but with the update, the space is larger than 50M. The tutorial on Alibaba Cloud function deployment has not been updated. to this point. I don't know exactly how to operate it. The main reason is that I don’t have time to fiddle with it later, so I gave up this method.

**Comment by lizheming** [comment, marked as answer]
https://v2ex.com/t/978589#reply7
